src/cgame/rocket/rocket.cpp

```diff
@@ -578,10 +579,10 @@ Rml::String Rocket_QuakeToRML( const char *in, int parseFlags = 0 )
 					out.append( spanstr );
 					spanHasContent = true;
 				}
-				out.append( token.Begin(), token.Size() );
+				out.append( text.begin(), text.size() );
 			}
 		}
-		else if ( token.Type() == Color::Token::TokenType::COLOR )
+		else // token.Type() == Color::Token::TokenType::COLOR
```

Comment:
why comment?

Reply:
I had an awkward if condition of type != COLOR to avoid diffs. I added another commit to swap the if-else branches.